<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFitting</class>
 <widget class="QWidget" name="DlgFitting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Surface fitting</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/cmiss_icon.ico</normaloff>:/cmiss_icon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="15">
    <widget class="ZincWidget" name="_zincWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnLoadData">
     <property name="text">
      <string>Load data</string>
     </property>
    </widget>
   </item>
   <item row="1" column="8" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>59</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="11">
    <widget class="QRadioButton" name="rbReference">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reference model</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="14">
    <widget class="QPushButton" name="btnQuit">
     <property name="text">
      <string>&amp;Quit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnLoadMesh">
     <property name="text">
      <string>Load mesh</string>
     </property>
    </widget>
   </item>
   <item row="1" column="12">
    <widget class="QRadioButton" name="rbFitted">
     <property name="font">
      <font>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Fitted model</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="13">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="btnSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnSelFaces">
     <property name="text">
      <string>Sel faces</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="btnSelData">
     <property name="text">
      <string>Sel data</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="btnFit">
     <property name="text">
      <string>Fit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="btnProject">
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZincWidget</class>
   <extends>QWidget</extends>
   <header>zincwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnQuit</sender>
   <signal>clicked()</signal>
   <receiver>DlgFitting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
